jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextHandler.java

```diff
@@ -290,13 +287,6 @@ public ServletContextHandler(String contextPath, SessionHandler sessionHandler,
         relinkHandlers();
     }
 
-    @Override
-    public void setTempDirectory(File tempDirectory)
-    {
-        super.setTempDirectory(tempDirectory);
-        setAttribute(TEMPDIR, getTempDirectory());
```

Comment:
Where does this spec attribute get set now?

Reply:
I've mistakenly moved that attribute from context to request. Let me move it back to where it belongs.